superset/annotation_layers/annotations/commands/exceptions.py

```diff
@@ -68,5 +64,5 @@ class AnnotationUpdateFailedError(CreateFailedError):
     message = _("Annotation could not be updated.")
 
 
-class AnnotationDeleteFailedError(CommandException):
```

Comment:
This illustrates a great example of the benefits of condensing logic. The bulk exceptions:

1. Is subclassed from  `DeleteFailedError`
2. Uses language of the form `Annotations could not be deleted.`

whereas the non-bulk exceptions:

1. Is subclassed from `CommandException`.
2. Uses language of the form  `Annotation delete failed.`

Here the legacy bulk logic seems more correct in terms of consistency with how create, update, etc. exceptions are defined.